read-excel/sheet.hpp

```diff
@@ -314,6 +320,21 @@ Sheet::handleLabelSST( Record & record )
 	m_cells[ row ][ column ].setData( m_sst[ sstIndex ] );
 }
 
+inline void
+Sheet::handleLabel( Record & record )
+{
+	int16_t row = 0;
+	int16_t column = 0;
+
+	record.dataStream().read( row, 2 );
+	record.dataStream().read( column, 2 );
+	record.dataStream().seek( 2, Excel::Stream::FromCurrent );
+	std::wstring data = loadString( record.dataStream(), record.borders(), 1 );
+
+	initCell( row, column );
+
+	m_cells[ row ][ column ].setData( data );
+}
 

```

Comment:
And now we need similar to `old.xls` test file, that will be BIFF8 but with `LABEL` record. `old.xls` is good, but name it as `strange.xls`, for example. But in `old.xls` I saw ugly string, like `????? / ???? ....` Could you find the same file but with good string in `LABEL` record to test it? In a few words: we need to test new functionality. Yes, yes, I know, what I said in previous PR, but now we need it.